src/components/Calculator.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Button from './Button';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Calculator extends React.Component {
+  render() {
+    return (
+      <div className="calculator">
```

Comment:
- [ ] The `Calculator.js` component should match the following given design:
[https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/images/calculator.png](image)

Reply:
Design looks okey